### DISCUSSION TITLE
Usecase: Annotating and discussing papers or sections of books

### DISCUSSION THREAD

**Discussion by wsdookadr** [excalidraw/excalidraw#7264] [General]
In this thread, I'll describe a workflow for a specific usecase of Excalidraw, namely that of talking about a paper. Excalidraw is a great communication tool for this purpose.

I've been using it with a friend to discuss some papers. We typically take some screenshot images of certain parts of a paper which we then upload to a canvas.
Then we do the following:
- we write on top of the images
- we point arrows at specific content or regions in the images and annotate them with additional comments
- we highlight parts of the images (especially definitions and various identities)
- we annotate with text or make small doodles
- we draw some arrows between existing diagrams in the images to some portion of text

This develops over the course of the conversation in a free-form way. It would definitely be nice to have MathJax Support #5890 and of course if #7161 CRDTs were added those would be a great addition too.

Another thing which would be interesting is uploading a PDF, have Excalidraw split it into pages and convert the pages to images and pre-populate the Canvas with them, because currently that's what we do manually. 

It would also be nice to be able to make some small experiments while we read the paper. Of course we can open a separate Jupyter and start writing some snippet there. If Excalidraw offered a way to insert an editable piece of code just like Jupyter Cell, that we can move around and fold/unfold, that would also be amazing. The part that Excalidraw would have to do in this case would be to just send the code off to a Jupyter Notebook/Lab or to a Sagemath instance.

**Comment by x3oo** [comment]
I am using "explain everything whiteboard" to teach online. This usecase is the most important for teaching/tutoring online.
